core/src/components/picker-column-option/picker-column-option.tsx

```diff
@@ -50,19 +65,17 @@ export class PickerColumnOption implements ComponentInterface {
   }
 
   render() {
-    const { value, disabled, ariaLabel } = this;
+    const { color, value, disabled, ariaLabel } = this;
+    const mode = getIonMode(this);
 
     return (
-      <Host>
-        <button
-          tabindex="-1"
-          aria-label={ariaLabel}
-          class={{
-            'picker-opt': true,
-            'picker-opt-disabled': !!disabled,
-          }}
-          disabled={disabled}
-        >
+      <Host
+        class={createColorClasses(color, {
+          [mode]: true,
+          ['option-disabled']: disabled,
```

Comment:
Correct, picker-column is in charge of adding that.

I opted to change the name since `picker-item-active` doesn't align with the component name of `ion-picker-column-option` (item vs option). The `PICKER_ITEM_ACTIVE_CLASS` usage will be cleaned up in https://github.com/ionic-team/ionic-framework/pull/28622